installer/templates/new/jobs/text.exs

```diff
@@ -3,5 +3,10 @@ use Kitto.Job.DSL
 job :text, every: {4, :seconds} do
   phrases = ["This is your shiny new dashboard", "Built on the Kitto Framework"]
 
-  broadcast! :text, %{text: (phrases |> Enum.shuffle |> Enum.take(1))}
+  broadcast! :text, 
+    %{
+      text: (phrases |> Enum.shuffle |> Enum.take(1)),
+      moreinfo: "This is where you can put dynamic text",
+      specialStyle: "widget-text-custom"
```

Comment:
I added this as a way that you could dynamically alter the widget style based on logic in your job. For example if it's alert text you could change the background to red or something like that.